setup.py

```diff
@@ -5,40 +5,23 @@
 from setuptools import find_packages
 from setuptools import setup
 
-
-long_desc = '''
-This library helps you deal with boolean expressions and algebra with variables
-and the boolean functions AND, OR, NOT.
-
-You can parse expressions from strings and simplify and compare expressions.
-You can also easily create your custom algreba and mini DSL and create custom
-tokenizers to handle custom expressions.  
-
-For extensive documentation look either into the docs directory or view it online, at
-https://booleanpy.readthedocs.org/en/latest/
-
-https://github.com/bastikr/boolean.py
-
-Copyright (c) 2009-2017 Sebastian Kraemer, basti.kr@gmail.com and others
-
-Released under revised BSD license.
-'''
-
+with open('README.rst') as readme:
+    long_description = readme.read()
 
 setup(
     name='boolean.py',
     version='3.4',
-    license='revised BSD license',
+    license='Simplified BSD license',
     description='Define boolean algebras, create and parse boolean expressions and create custom boolean DSL.',
-    long_description=long_desc,
+    long_description=long_description,
     author='Sebastian Kraemer',
     author_email='basti.kr@gmail.com',
     url='https://github.com/bastikr/boolean.py',
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    test_loader='unittest:TestLoader',
```

Comment:
The previous test suite could run equally well with or without pytest. Any specil reason to make py.test a hard requirement for tests?

Reply:
(I know this is a _very_ old PR, but...)

The use of `@pytest` decorators makes pytest a hard requirement, full stop. (And I agree with @alisianoi: it's better _not_ to support multiple test frameworks, which lead to everyone executing the tests in subtly different ways. Do that, you've stopped testing your code; now you're compatibility-testing the frameworks instead.)

`pytest-runner` [is deprecated](https://pypi.org/project/pytest-runner/#description). (As are any use of `test_runner`, `test_suite`, `tests_require`, etc. in `setup.py`. `setup.py` _as a whole_ is deprecated, of course, but its test args are "super double deprecated".) These days, `tox` would be a better choice as test runner.